numpy/core/src/multiarray/sequence.c

```diff
@@ -50,9 +50,32 @@ array_contains(PyArrayObject *self, PyObject *el)
     return ret;
 }
 
+static PyObject*
+array_concat(PyObject *self, PyObject *other) {
+    PyObject *combine, *res;
+    Py_INCREF(self);
+    Py_INCREF(other);
+    //Check if we are trying to concatenate a non-ndarray with a ndarray
+    if(!PyArray_Check(other)) {
+        PyErr_SetString(PyExc_ValueError,
+                        "Can only concatenate a ndarray with a ndarray");
+        return NULL;
+    }
+    //Convert the objects to a tuple , pass PyArrayConcatenate
+    combine = PyTuple_Pack(2, self, other);
+    if (combine == NULL) {
+        PyErr_SetString(PyExc_ValueError,
+            "Unable to create Tuple from ndarray Objects");
+        return NULL;
+    }
+    res = PyArray_Concatenate(combine,0);
+    Py_DECREF(combine);
+    return res;
+}
+
 NPY_NO_EXPORT PySequenceMethods array_as_sequence = {
     (lenfunc)array_length,                  /*sq_length*/
-    (binaryfunc)NULL,                       /*sq_concat is handled by nb_add*/
+    (binaryfunc)array_concat,                       /*sq_concat is handled by nb_add*/
```

Comment:
I think this comment should say
```suggestion
    (binaryfunc)array_concat,                       /*sq_concat, for operator.concat*/
```